src/main/java/cn/nukkit/player/Player.java

```diff
@@ -316,15 +317,33 @@ public boolean isBanned() {
     }
 
     @Override
+    @Deprecated
     public void setBanned(boolean value) {
```

Comment:
I don't think there is any point in deprecating this method as this is a major version change. Removing it altogether would be okay.

Reply:
Oh sry, i tried somethings with the code :).

Reply:
what is with BanEntry and BanList? Can we rewrite the classes?